test/martha/martha_v3.test.js

```diff
@@ -815,7 +815,7 @@ test.serial('martha_v3 fails if something times out before trying to fetch a sig
     );
 }));
 
-test.serial('martha_v3 succeeds if an error is encountered while fetching a signed URL', async (t) => {
```

Comment:
I think we still want this test, though I agree that we can add another test for the S3 case.

Reply:
In its current state the Martha v3 code does not attempt to fetch signed URLs for non-S3 objects, so there can't be errors in that process.

Reply:
Ugh. I guess you're right.

I'm frustrated because I put a bunch of work into adding the behavior that this test checks. As the test is now, I'm afraid all of that work could be undone and all the tests will still pass.

One of these days, I'd like to rewrite these tests to check for the expected behavior based on different permutations of `DrsType` instead of specifically for the projects we support. (We probably want some tests like that too, but we could be testing the base capabilities of Martha separately.)